core/frontend/src/extension/providers/ExtensionServiceClient.ts

```diff
@@ -72,12 +72,13 @@ export class ExtensionClient {
   }
 
   /**
-   * Gets information on extensions. If extensionName is undefined, will return all extensions in the context.
+   * Gets information on extensions. If extensionName is undefined, will return all extensions in the iTwinId.
    * If it's defined, will return all versions of that extension.
-   * @param iTwinId Context Id
+   * If iTwinId is undefined, will default to public.
    * @param extensionName Extension name (optional)
+   * @param iTwinId iTwin Id (optional)
    */
-  public async getExtensions(accessToken: AccessToken, iTwinId: string, extensionName?: string): Promise<ExtensionMetadata[]> {
+  public async getExtensions(accessToken: AccessToken, extensionName?: string, iTwinId = "00000000-0000-0000-0000-000000000000"): Promise<ExtensionMetadata[]> {
```

Comment:
can use Guid.empty from the core-bentley package

Reply:
Ah perfect! I wasn't aware of that